charts/device-plugin-operator/templates/operator.yaml

```diff
@@ -462,7 +470,11 @@ spec:
         - --logtostderr=true
         - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
         - --v=10
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1
+        {{- if .Values.privateRegistry.registryUrl }}
+        image: "{{ .Values.privateRegistry.registryUrl }}/{{ .Values.kubeRbacProxy.image.hubRepo }}:{{ .Values.kubeRbacProxy.image.tag }}"
+        {{- else }}
+        image: "{{ .Values.kubeRbacProxy.image.hub }}/{{ .Values.kubeRbacProxy.image.hubRepo }}/kube-rbac-proxy:{{ .Values.kubeRbacProxy.image.tag }}"
```

Comment:
mismatch with `kubeRbacProxy` vs `kube-rbac-proxy`?

Reply:
Do we need the "privateRegistryUrl"? User could just set `manager.image.hub` or `kubeRbacProxy.image.hub` as he/she pleases?

Reply:
@tkatila is right and I prefer that option. it follows the same pattern as with the CRs